src/core/hooks/usePasteBlocks.ts

```diff
@@ -61,7 +61,7 @@ export const usePasteBlocks = (): {
     pasteBlocks: useCallback(
       (newParentId: string | string[]) => {
         if (!isEmpty(copiedBlockIds)) {
-          duplicateBlocks(copiedBlockIds, newParentId);
+          duplicateBlocks(copiedBlockIds, newParentId[0]);
```

Comment:
what if the newParentId is string? 
It should handle both cases.

Reply:
Yeah resolved it